math/OpenBLAS/Portfile

```diff
@@ -260,61 +187,30 @@ platform darwin 8 {
-                ${destroot}-${arch}${prefix}/include/cblas_openblas.h
-            #Correct library name
-            system "install_name_tool -id \
-                ${prefix}/lib/libopenblas-r1.dylib \
-                ${destroot}-${arch}${prefix}/lib/libopenblas-r1.dylib"
-            delete file ${destroot}-${arch}${prefix}/lib/cmake
-            if {[variant_isset lapack]} {
-                #Install lapacke headers too
-                set LAPACKE_DIR [exec /bin/ls ${worksrcpath}/lapack-netlib | \
-                                     grep -i lapacke]
-                xinstall -m 755 ${worksrcpath}-${arch}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_config.h \
-                    ${destroot}-${arch}${prefix}/include/lapacke_config.h
-                xinstall -m 755 ${worksrcpath}-${arch}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_mangling.h \
-                    ${destroot}-${arch}${prefix}/include/lapacke_mangling.h
-                xinstall -m 755 ${worksrcpath}-${arch}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_utils.h \
-                    ${destroot}-${arch}${prefix}/include/lapacke_utils.h
-                xinstall -m 755 ${worksrcpath}-${arch}/lapack-netlib/${LAPACKE_DIR}/include/lapacke.h \
-                    ${destroot}-${arch}${prefix}/include/lapacke.h
-            }
+post-destroot {
+    #For compatibility, put header files in ${prefix}/include
+    foreach h [glob -tails -directory "${destroot}${prefix}/include/openblas" *.h] {
+        if {${h} eq "cblas.h"} {
+            ln -s cblas/${h} ${destroot}${prefix}/include/cblas_openblas.h
+        } else {
+            ln -s cblas/${h} ${destroot}${prefix}/include/${h}
         }
     }
-} else {
-    post-destroot {
-        #Avoid conflict with Atlas
-        move ${destroot}${prefix}/include/cblas.h \
-            ${destroot}${prefix}/include/cblas_openblas.h
-        #Correct library name
-        system "install_name_tool -id \
-            ${prefix}/lib/libopenblas-r1.dylib \
-            ${destroot}${prefix}/lib/libopenblas-r1.dylib"
-        delete file ${destroot}${prefix}/lib/cmake
-        if {[variant_isset lapack]} {
-            #Install lapacke headers too
-            set LAPACKE_DIR [exec /bin/ls ${worksrcpath}/lapack-netlib | \
-                                 grep -i lapacke]
-            xinstall -m 755 ${worksrcpath}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_config.h \
-                ${destroot}${prefix}/include/lapacke_config.h
-            xinstall -m 755 ${worksrcpath}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_mangling.h \
-                ${destroot}${prefix}/include/lapacke_mangling.h
-            xinstall -m 755 ${worksrcpath}/lapack-netlib/${LAPACKE_DIR}/include/lapacke_utils.h \
-                ${destroot}${prefix}/include/lapacke_utils.h
-            xinstall -m 755 ${worksrcpath}/lapack-netlib/${LAPACKE_DIR}/include/lapacke.h \
-                ${destroot}${prefix}/include/lapacke.h
```

Comment:
This part regarding Lapacke headers has been removed. Is it on purpose of has it been forgotten?

Reply:
I believe the Lapacke header files are installed by the port itself in [Makefile.install](https://github.com/xianyi/OpenBLAS/blob/b89fb708caa5a5a32de8f4306c4ff132e0228e9a/Makefile.install#L74).